devices/aldi-lightway-smart-al2017-tgts-downlight.json

```diff
@@ -1,12 +1,14 @@
 {
-	"manufacturer": "Aldi Lightway",
-	"name": "SMART-AL2017-TGTS Downlight",
+	"manufacturer": "Aldi",
+	"name": "SMART-AL2017-TGTS Lightway 8W Downlight",
```

Comment:
Should probably consider renaming the base file when we change the mfg/name as well.